sql/driver/wire.go

```diff
@@ -48,6 +59,12 @@ func (d Datum) String() string {
 		return string(t)
 	case *string:
 		return *t
+	case *Datum_Time:
+		timeVal := time.Unix((*t).Sec, int64((*t).Nsec)).UTC()
+		if timeVal.Equal(timeVal.Truncate(24 * time.Hour)) {
```

Comment:
I think this is trying to identify when a `Datum_Time` refers to a `Date` vs a `Timestamp`. Seems a bit hokey to be doing it this way. Is there a reason we can't always just output as a timestamp?